Packs/Core/Layouts/layoutscontainer-Default_platform_alerts.json

```
@@ -0,0 +1,151 @@
+{
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
+                "name": "Legacy Summary",
+                "type": "summary"
+            },
+            {
+                "id": "warRoom",
+                "name": "War Room",
+                "type": "warRoom"
+            },
+            {
+                "id": "workPlan",
+                "name": "Work Plan",
+                "type": "workPlan"
+            },
+            {
+                "id": "canvas",
+                "name": "Canvas",
+                "type": "canvas",
+                "hidden": false
+            }
+        ]
+    },
+    "group": "incident",
+    "id": "Default Platform Alerts",
+    "name": "Default Platform Alerts",
```

Replace all occurrences of Alerts with 
Issues```suggestion
    "name": "Default Platform Issues",
```